docroot/themes/humsci/humsci_basic/src/scss/utilities/_lists.scss

```diff
@@ -3,6 +3,12 @@
     div div:not([class]) {
       @include hb-category-item;
     }
+
+    & > a:not([class]),
+    & > div > a:not([class]) {
+      // TODO this isn't perfect, link hover style gets applied to the psuedo element
```

Comment:
what do you think about this @kaseybon ? I couldn't think of a perfect workaround